File: baidu-search/src/main/java/io/github/llmagentbuilder/tool/baidusearch/BaiduSearchConfig.java
package io.github.llmagentbuilder.tool.baidusearch;

public class BaiduSearchConfig {

  private int numberOfItems = 10;

  public int getNumberOfItems() {
    return numberOfItems;
  }

  public void setNumberOfItems(int numberOfItems) {
    this.numberOfItems = numberOfItems;
  }

  @Override
  public String toString() {
    return "BaiduSearchConfig{" +
        "numberOfItems=" + numberOfItems +
        '}';
  }
}

File: baidu-search/src/main/kotlin/io/github/llmagentbuilder/tool/baidusearch/BaiduSearchConfig.kt
package io.github.llmagentbuilder.tool.baidusearch

data class BaiduSearchConfig(val numberOfItems: Int? = null)

data class BaiduSearchRequest(val keyword: String? = null)

data class BaiduSearchResultItem(
    val title: String? = null,
    val url: String? = null
)

data class BaiduSearchResponse(val items: List<BaiduSearchResultItem>? = null)